docs/en/ingest-management/commands.asciidoc

```diff
@@ -227,6 +240,11 @@ Private key to use for exposed {fleet-server} HTTPS endpoint.
 `--fleet-server-cert-key-passphrase <string>`::
 Path to passphrase file for decrypting {fleet-server}'s private key if an encrypted private key is used.
 
+`--fleet-server-client-auth <string>`::
+One of "none", "optional", or "required". Defaults to "none". {fleet-server}'s client_authentication option
+for client mTLS connections. If "optional", or "required" are specified, client certificates are verified
```

Comment:
```suggestion
One of `none`, `optional`, or `required`. Defaults to `none`. {fleet-server}'s `client_authenticatio` option
for client mTLS connections. If `optional`, or `required` is specified, client certificates are verified
```
Just switching quotes to backticks, since use the latter for setting values in most of the docs.